Kudu.Core/Deployment/Generator/ExternalCommandFactory.cs

```diff
@@ -168,9 +167,9 @@ private static string KreBitness
                 var bitness = System.Environment.GetEnvironmentVariable(WellKnownEnvironmentVariables.SiteBitness);
                 if (bitness == null)
                 {
-                    return System.Environment.Is64BitProcess ? "amd64" : "x86";
+                    return System.Environment.Is64BitProcess ? "X64" : "x86";
```

Comment:
Only if `WellKnownEnvironmentVariables.SiteBitness` is null. kudu's `w3wp.exe` will be the same bitness as the main site's process, but that's not true for `kudu.exe`. If deployment is happening in `kudu.exe`, then that environment variable won't be null, else we are running deployment in `w3wp.exe`

But this actually reminds me I should fix that environment variable as well to be X64 rather than Amd64